meta_client_v2/src/types.rs

```diff
@@ -186,6 +186,21 @@ pub struct CloseCmd {
     pub shard_ids: Vec<ShardId>,
 }
 
+#[derive(Debug, Clone)]
+pub struct CreateTableCmd {
+    pub schema_name: String,
+    pub name: String,
+    pub shard_id: ShardId,
+    pub schema_id: SchemaId,
+    pub id: TableId,
+}
+
+#[derive(Debug, Clone)]
+pub struct DropTableCmd {
+    pub schema_name: String,
+    pub name: String,
+}
```

Comment:
I think we also need a `CloseTableCmd` to only close an opened table from one instance without dropping it?

Reply:
Actually,  `Close(Shards)Cmd` and `Open(Shards)Cmd` exist rather than `CloseTableCmd` and `OpenTableCmd`.